### ISSUE TITLE
e2e setup

### ISSUE THREAD

**Issue by greg-nagy** [tallycash/extension#1733]

**Comment by greg-nagy** [comment]
I feel that the spike part is done, and we should transition into work breakdown after some handoff.

Not closing yet, because the information is not yet collected into a coherent place. 

Summary 
- We can modify synpress to install tally and use it as a tool to base our e2e tests off @greg-nagy 
- We can use browserstack to run our tests on even from the CI 
- We can use OCR + selenium + browserstack to handle native actions like clicking the browser action button @puppycodes

**Comment by puppycodes** [comment]
Yes, theres actually another more simple framework I discovered that does exactly this that I have totally setup and is ready to go. Check your email for an invite https://rainforestqa.com

**Comment by puppycodes** [comment]
This is going to be taken over by Idan moving into the next sprint

**Comment by 0xDaedalus** [comment]
Spike is done
